### PR TITLE
SPARKC-318: Add tests for fix mapping a column with null value

### PR DESCRIPTION
**Pull request by rushton**
Adds tests to #1008

### CODE REVIEW

File: spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TypeConverter.scala [hunk 1]
```diff
@@ -60,7 +60,8 @@ trait NullableTypeConverter[T <: AnyRef] extends TypeConverter[T] {
 
 /** Chains together several converters converting to the same type.
   * This way you can extend functionality of any converter to support new input types. */
-class ChainedTypeConverter[T](converters: TypeConverter[T]*) extends TypeConverter[T] {
+class ChainedTypeConverter[T](converters: TypeConverter[T]*) extends NullableTypeConverter[T] {
+  def testAnyVal[T](x: T)(implicit evidence: T <:< AnyVal = null) = evidence != null
```
Comment (RussellSpitzer) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TypeConverter.scala]: What is this for?

Comment (rushton) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TypeConverter.scala]: I have no idea, I searched for scala documentation for what this actually does but came up empty. @delyand will have the best explanation as to what it does

Comment (RussellSpitzer) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/TypeConverter.scala]: It looks like a check for whether or not the value is "AnyVal" I'm thinking but I don't know where it's used

**Comment by bcantoni** [comment]
@rushton we're rolling out a Contributor License Agreement for the Spark Cassandra Connector and need your signature to be able to use your contributions.

Please [follow instructions](http://spark-cassandra-connector-cla.datastax.com/) to sign, the whole process is completely electronic and will only take minutes.

Don't hesitate to <a href="mailto:cla@datastax.com?subject=Spark%20Cassandra%20Connector%20CLA">email cla at datastax.com</a> if you have any additional questions.

**Comment by RussellSpitzer** [comment]
Merged in https://github.com/datastax/spark-cassandra-connector/pull/1158
